flow/connectors/postgres/qvalue_convert.go

```diff
@@ -577,6 +581,25 @@ func (c *PostgresConnector) parseFieldFromPostgresOID(
 			}
 			return qvalue.QValueArrayEnum{Val: a}, nil
 		}
+	case qvalue.QValueKindArrayNumeric:
+		if items, ok := value.([]any); ok {
+			a := make([]string, len(items))
+			success := true
+			for i, item := range items {
+				var numeric pgtype.Numeric
+				var ok bool
+				if numeric, ok = item.(pgtype.Numeric); !ok {
+					success = false
+					break
+				}
+				bytes, _ := numeric.MarshalJSON()
+				a[i] = shared.UnsafeFastReadOnlyBytesToString(bytes)
+			}
+			if success {
+				return qvalue.QValueArrayString{Val: a}, nil
```

Comment:
```suggestion
				return qvalue.QValueArrayNumeric{Val: a}, nil
```
code is in transition at the moment to have full numeric support & that this'll be decimal.Decimal slice anyways?